external/twitter/src/main/scala/org/apache/spark/streaming/twitter/TwitterUtils.scala

```diff
@@ -53,7 +67,7 @@ object TwitterUtils {
    * @param jssc   JavaStreamingContext object
    */
   def createStream(jssc: JavaStreamingContext): JavaReceiverInputDStream[Status] = {
-    createStream(jssc.ssc, None)
+    createStream(jssc.ssc, None) 
```

Comment:
There's a couple spurious changes adding whitespace at the ends of lines. Not really worth fixing for its own sake but check the diff to see if you can fix it if you make another commit.